_sass/base/blocks/_school.scss

```diff
@@ -60,7 +60,7 @@
   //   padding-top: $base-padding-lite;
   // }
 
-  @include respond-to(medium-up) {
+  @include respond-to(large-up) {
```

Comment:
Expected item on line 63 to appear before line 35. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets